src/styles/_catalog.scss

```
@@ -1,22 +1,29 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
-  margin: 0 auto;
+  display: grid;
+  grid-template-columns: repeat(1, $card-width);
```

no need to repeat one column
```suggestion
  grid-template-columns: $card-width;
```